docs/widgetbook-cloud/reviews.mdx

```diff
@@ -1,124 +1,58 @@
 # Widgetbook Cloud Review
```

Comment:
This file lacks a bit of structure.

I'd separate the Regression Detector and Figma x Flutter Comparison straight in the beginning with an agenda, like:

1. Regression Detector
2. Figma x Flutter Comparison